doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -443,13 +443,17 @@ Modem libraries
 
   * Updated:
 
+    * The default network mode to :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS`.
+    * The default LTE mode preference to :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE_LTE_M_PLMN_PRIO`.
     * The :c:func:`lte_lc_init` function is deprecated.
     * The :c:func:`lte_lc_deinit` function is deprecated.
       Use the :c:func:`lte_lc_power_off` function instead.
     * The :c:func:`lte_lc_init_and_connect` function is deprecated.
       Use the :c:func:`lte_lc_connect` function instead.
     * The :c:func:`lte_lc_init_and_connect_async` function is deprecated.
       Use the :c:func:`lte_lc_connect_async` function instead.
+    * The :kconfig:option:`CONFIG_LTE_NETWORK_USE_FALLBACK` Kconfig option is deprecated.
```

Comment:
I would move this to 3rd bullet below network mode and LTE mode preference as this is tight to those changes.